src/corePlugins.js

```diff
@@ -1782,18 +1784,42 @@ export let corePlugins = {
           shadow: (value) => {
             value = transformValue(value)
 
+            let ast = parseBoxShadowValue(value)
+            for (let shadow of ast) {
+              // Don't override color if the whole shadow is a variable
+              if (shadow.x === undefined || shadow.y === undefined) {
+                continue
+              }
```

Comment:
Should we encode this information in each shadow itself? Like a `isValid` flag or something?
Not a requirement, but I noticed the same logic in multiple places.

Reply:
Yeah I agree or even have the parse function return an error or something if it doesn’t parse a proper value. Moar tuples? Is this Elm?